joss/paper.md

```diff
@@ -86,39 +86,37 @@ ax, hm = CairoMakie.heatmap(fig[2,1], reg_sens.standard_regression,
                             xticklabelsvisible = false,
                             title = "Standard regression"))
 Colorbar(fig[2, 2], hm)
+Label(fig[3, :], "Regression based GSA for Lotka Volterra model")
 fig
 ```
 
-#### Regression based GSA for Lotka Volterra model
-
-![heatmapreg](https://user-images.githubusercontent.com/23134958/127019339-607b8d0b-6c38-4a18-b62e-e3ea0ae40941.png)
+![Heatmap of partial correlation and standard regression coefficients for the maximum of the prey population and the mean of the predator population from the Lotka Volterra model. The first column in each of the heatmaps corresponds to the maximum of the predator population and the second column corresponds the mean of the prey population. The rows follow the order of the parameters in the vector, first row being `p[1]` second `p[2]` and so on.](https://user-images.githubusercontent.com/23134958/177251367-9c93a219-8871-4734-8e3f-b6e3f6b6d90a.png)
 
 Next, the Morris method is used and results are visualized as a scatter plot.
 
 ```julia
 using StableRNGs
 _rng = StableRNG(1234)
 morris_sens = gsa(f1, Morris(), bounds, rng = _rng)
-fig = Figure(resolution = (300, 200))
+fig = Figure(resolution = (400, 300))
 scatter(fig[1,1], [1,2,3,4], morris_sens.means_star[1,:],
         color = :green, axis = (xticksvisible = false,
         xticklabelsvisible = false, title = "Prey (Morris)",))
 scatter(fig[1,2], [1,2,3,4], morris_sens.means_star[2,:],
         color = :red, axis = (xticksvisible = false,
         xticklabelsvisible = false, title = "Predator (Morris)",))
+Label(fig[2, :], "Morris method GSA for Lotka Volterra model")
 fig
 ```
 
-#### Morris method GSA for Lotka Volterra model
-
-![morrisscat](https://user-images.githubusercontent.com/23134958/127019346-2b5548c5-f4ec-4547-9f8f-af3e4b4c317c.png)
+![Scatter plot of `μ*` for the mean of the prey population and maximum of the predator population from the Morris method. The first plot corresponds to the mean of the prey population and the second plot corresponds to the maximum of the predator population. Each of the four parameters occur in both plots as one scatter point with `μ*` on the y-axis and index of the parameter corresponding to its order on the x-axis.](https://user-images.githubusercontent.com/23134958/177251158-69a6e8b8-0a46-4998-b9f2-106886827176.png)
```

Comment:
First, second, etc. doesn't have a clear meaning. "Left: the Morris indices for the mean of the prey population". Top left, bottom right, etc. If you really took the time you'd add (A) etc. to the plots for naming, but left right top bottom should be fine.